src/components/StaffComments/StaffComments.vue

```diff
@@ -1,91 +1,92 @@
 <template>
-  <!-- NB: attach the menu to its button so we can unit test it -->
-  <v-menu
-    bottom
-    v-model="showComments"
-    attach="#comments-button"
-    :close-on-click="false"
-    :close-on-content-click="false"
-  >
-    <!-- the button -->
-    <template v-slot:activator="{ on, attrs }">
-      <v-btn
-        small text color="primary"
-        id="comments-button"
-        v-bind="attrs"
-        v-on="on"
-      >
-        <v-icon medium>mdi-comment-text-outline</v-icon>
-        <span>{{numComments}}</span>
-      </v-btn>
-    </template>
-
-    <!-- the menu (dialog) -->
-    <!-- NB: stop click events from propagating up to comments-btn -->
-    <v-card flat id="staff-comment-container" class="px-8 py-6" @click.stop :ripple="false">
-      <v-card-title class="d-flex justify-space-between pa-0">
-        <div>
-          <v-icon medium color="primary">mdi-comment-text-outline</v-icon>
+  <div class="d-inline" id="staff-comments">
```

Comment:
The big change here is that I now attach the v-menu to a component div instead of its v-btn. The latter made every child look and act like a button, which I sort of worked around with `@click.stop` and `:ripple="false"` but there was other bad behaviour.
All better now :)